libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2495,11 +2495,10 @@ string YulUtilFunctions::readFromMemoryOrCalldata(Type const& _type, bool _fromC
 	});
 }
 
-string YulUtilFunctions::revertReasonIfDebug(RevertStrings revertStrings, string const& _message)
+string YulUtilFunctions::revertReasonIfDebugAssembly(RevertStrings _revertStrings, std::string const& _message)
 {
-	if (revertStrings >= RevertStrings::Debug && !_message.empty())
-	{
-		Whiskers templ(R"({
+	Whiskers templ(R"({
+		<?debugAndMessage>
```

Comment:
Just a suggestion but I think that being able to see the refactor that remodels this function to use `<?debugAndMessage>` separately from the one that that renames frunctions and introduces `revertReasonIfDebugFunction()` would make this PR slightly easier to review (not that it's hard to review, but anything that makes reviews easier is a good thing). If they were in separate commits I could easily focus on just one at a time.